[2023-06-11T08:24:26.314+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_pythonoperator_18.get_name manual__2023-06-11T08:22:13.204101+00:00 [queued]>
[2023-06-11T08:24:27.469+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_pythonoperator_18.get_name manual__2023-06-11T08:22:13.204101+00:00 [queued]>
[2023-06-11T08:24:27.735+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-11T08:24:29.915+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): get_name> on 2023-06-11 08:22:13.204101+00:00
[2023-06-11T08:24:30.141+0000] {standard_task_runner.py:57} INFO - Started process 970 to run task
[2023-06-11T08:24:30.420+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_pythonoperator_18', 'get_name', 'manual__2023-06-11T08:22:13.204101+00:00', '--job-id', '304', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag.py', '--cfg-path', '/tmp/tmp8swg17i6']
[2023-06-11T08:24:30.421+0000] {standard_task_runner.py:85} INFO - Job 304: Subtask get_name
[2023-06-11T08:24:32.490+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_pythonoperator_18.get_name manual__2023-06-11T08:22:13.204101+00:00 [running]> on host 492037cdd057
[2023-06-11T08:24:35.004+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Raja' AIRFLOW_CTX_DAG_ID='dag_pythonoperator_18' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-06-11T08:22:13.204101+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-11T08:22:13.204101+00:00'
[2023-06-11T08:24:35.286+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-06-11T08:24:35.960+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_pythonoperator_18, task_id=get_name, execution_date=20230611T082213, start_date=20230611T082426, end_date=20230611T082435
[2023-06-11T08:24:36.284+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-11T08:24:37.328+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
